modules/test/playwright/tests/object-web/objectLayout.spec.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {editObjectDefinitionPagesTest} from '../../fixtures/editObjectDefinitionPagesTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {objectPagesTest} from '../../fixtures/objectPagesTest';
+import getRandomString from '../../utils/getRandomString';
+import getRandomObjectFieldText from './utils/getRandomObjectFieldText';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	editObjectDefinitionPagesTest,
+	featureFlagsTest({
+		'LPS-187854': true,
+	}),
+	loginTest(),
+	objectPagesTest
+);
+
+test.describe('Manage Object Layouts through the Object Layout tab', () => {
+	test('Can view all fields of an object when creating its layout', async ({
+		apiHelpers,
+		objectLayoutsPage,
+		page,
+	}) => {
+		const objectFields = getRandomObjectFieldText({
+			objectFieldsQuantity: 20,
+		});
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFields,
+				status: {code: 0},
+			});
+
+		const objectFieldLabelList = [];
+
+		const objectLayoutName = getRandomString();
+
+		objectFields.forEach((objectField) => {
+			objectFieldLabelList.push(objectField.label.en_US);
+		});
+
+		await objectLayoutsPage.goto(objectDefinition.name);
+
+		await objectLayoutsPage.createObjectLayout(objectLayoutName);
+
+		await objectLayoutsPage.openObjectLayout(objectLayoutName);
+
+		objectFieldLabelList.forEach((objectFieldLabel) => {
+			expect(
+				page
+					.frameLocator('iframe')
+					.getByRole('option', {name: objectFieldLabel})
+			).toBeVisible();
+		});
```

Comment:
Hey @vngoliveira, I saw that we have already made progress here, but I realized that we can improve even more. Avoiding the creation of a constant and eliminating the need for iteration in the list.

We can desestructure the label property from objectFields array, for exemple:

```
		objectFields.forEach(({label}) => {
			expect(
				page
					.frameLocator('iframe')
					.getByRole('option', {name: label.en_US})
			).toBeVisible();
		});
```